lepton/image.go

```diff
@@ -154,40 +154,45 @@ func addFilesFromPackage(packagepath string, m *Manifest) {
 func BuildPackageManifest(packagepath string, c *Config) (*Manifest, error) {
 	m := NewManifest()
 
-	m.program = c.Program
-	addFromConfig(m, c)
+	// Add files from package
+	addFilesFromPackage(packagepath, m)
 
+	m.program = c.Program
+	err := addFromConfig(m, c)
+	if err != nil {
+		return nil, errors.Wrap(err, 1)
+	}
 	if len(c.Args) > 1 {
 		if _, err := os.Stat(c.Args[1]); err == nil {
 			m.AddFile(c.Args[1], c.Args[1])
 		}
 	}
 
-	// Add files from package
-	addFilesFromPackage(packagepath, m)
 	return m, nil
 }
 
 func addFromConfig(m *Manifest, c *Config) error {
-
 	m.AddKernel(c.Kernel)
 	addDNSConfig(m, c)
 	addHostName(m, c)
 	addPasswd(m, c)
```

Comment:
this might be over-writing the stub etc/passwd nginx has for the nobody user - it's not a real user but nginx wants it - the error you are seeing here -> ```nginx: [emerg] getpwnam("nobody") failed``` suggests that it can't find the entry for it